From c974ee988f6136d26286140f0de134874aec2e9f Mon Sep 17 00:00:00 2001
From: nopdan <cxprcn@gmail.com>
Date: Fri, 27 Jun 2025 11:12:11 +0800
Subject: [PATCH] opencc

---
 CMakeLists.txt     | 2 +-
 src/CMakeLists.txt | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 1acb75a..450957a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -222,7 +222,7 @@ endif()
 
 add_subdirectory(src)
 add_subdirectory(doc)
-add_subdirectory(data)
+# add_subdirectory(data)
 add_subdirectory(test)
 
 ######## Testing
diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 75eda02..bc7191a 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -205,4 +205,4 @@ endif()
 
 # Subdir
 
-add_subdirectory(tools)
+# add_subdirectory(tools)
-- 
2.48.1.windows.1

